config/noi.json

```diff
@@ -0,0 +1,25 @@
+{
+    "feed_data": {
+        "system_information": {
+            "email": "info@carsharing.bz.it",
+            "feed_contact_email": "mail@leonard.io",
+            "language": "de",
+            "license_url": "https://www.govdata.de/dl-de/by-2-0",
+            "license_id": "DL-DE-BY-2.0",
+            "attribution_organization_name": "Konsortialgenossenschaft Car Sharing Südtirol Alto Adige",
+            "attribution_url": "https://www.carsharing.bz.it",
```

Comment:
According to their website, they recently rename to [www.alpsgo.it](http://www.alpsgo.it/)

Reply:
@hbruch Yes, they are changing the organization. But the feed is still related to the previous organization. Probably we will have to activate a new interface with the new organization since they will rely on another technological car sharing supplier

Reply:
Will the new organisation provide a GBFS feed themselves? That would be the best option and reduce the number of transformations we have to do.

Reply:
@leonardehrenfried It will always be like that: we get the data from the data supplier via custom API, we integrate it in the Open Data Hub, and then we expose the (standard) interfaces that are needed

Reply:
@rcavaliere another option would be, to convert proprietary APIs into GBFS (maybe the alpsgo.it provider is already available via `x2gbfs`) and implement once a GBFS2datahub importer. That way, NOI could on the fly provide the nowadays commonly used GBFS. And most sharing providers like TIER, BOLT, VOI provide them already themselves.

Reply:
Ok, I don't think there is anything left for me to do here.